plugins/flytekit-memray/setup.py

```diff
@@ -0,0 +1,37 @@
+from setuptools import setup
+
+PLUGIN_NAME = "memray"
+
+microlib_name = f"flytekitplugins-{PLUGIN_NAME}"
+
+plugin_requires = ["flytekit>=1.12.0", "memray"]
+
+__version__ = "0.0.0+develop"
+
+setup(
+    name=microlib_name,
+    version=__version__,
+    author="flyteorg",
+    author_email="admin@flyte.org",
+    description="This package enables memory profiling for tasks with memray",
+    namespace_packages=["flytekitplugins"],
+    packages=[f"flytekitplugins.{PLUGIN_NAME}"],
+    install_requires=plugin_requires,
+    license="apache2",
+    python_requires=">=3.8",
```

Comment:
we [recently dropped](https://github.com/flyteorg/flytekit/pull/2909) support for python 3.8. Can you also remove it from here?